package.json

```diff
@@ -9,11 +9,15 @@
   },
   "dependencies": {
     "axios": "^0.21.1",
+    "i18next-http-backend": "^1.2.2",
```

Comment:
why do we need to change this?

Reply:
this line is already added in upstream/main. I have not added it.